swan-lake/integration-tutorials/sending-a-message-to-a-service.md

```diff
@@ -214,15 +230,10 @@ bal run hospitalservice.jar
 
 #### Send a request
 
-Let's send a request to the service using cURL as follows.
+> **Note:** The healthcare service has to be running to send a request to it. If you have stopped the service, start it again.
```

Comment:
Why do we need to mention this explicitly?

Reply:
We don't need it